bin/v-add-user

```diff
@@ -35,11 +35,10 @@ source_conf "$HESTIA/conf/hestia.conf"
 is_user_free() {
 	# these names may cause issues with MariaDB/MySQL database names and should be reserved:
 	# sudo has been added due to Privilege escalation as sudo group has always sudo permission
-	check_sysuser=$(php -r '$reserved_names=array("aria", "aria_log", "mysql", "mysql_upgrade", "ib", "ib_buffer",
- "ddl", "ddl_recovery", "performance", "sudo"); if(in_array(strtolower($argv[1]), $reserved_names, true)){echo implode(", ", $reserved_names);}' "$user")
-	if [ -n "$check_sysuser" ]; then
-		check_result "$E_INVALID" "The user name '$user' is reserved and cannot be used. List of reserved names: $check_sysuser"
-		return
+	reserved_names=("aria" "aria_log" "mysql" "mysql_upgrade" "ib" "ib_buffer" "ddl" "ddl_recovery" "performance" "sudo")
+	if [[ "${reserved_names[@],,}" =~ "${user,,}" ]]; then
```

Comment:
Does not really work as expected... 

root@dev:~# v-add-user buffer 1234 info@hestiacp.com
Error: The user name 'buffer' is reserved and cannot be used. List of reserved names: aria aria_log mysql mysql_upgrade ib ib_buffer ddl ddl_recovery performance sudo

buffer or even pref should be allowed as user names..